server/async-worker/scraperQueue.js

```diff
@@ -1,8 +1,9 @@
 const Queue = require('bull');
-const { Mention, Company, User } = require("../models");
+const { Mention, Company, User, CompanyMentions } = require("../models");
+const UserCompanies = require("../models/userCompanies")
```

Comment:
You're welcome to include UserCompanies as part of the export object of the models directory, so you don't have to include it another line. Sorry I didn't do it myself; didn't think the object itself would actually end up being used. Glad to see the CompanyMentions thing working out for you too

Reply:
I have always been confused by this "export" thing, so I just did it the easy way...
And your code regarding CompanyMentions is absolutely correct, thank you~